pagopa/pn-frontend-e2e
# Pn 10344 10345

## Short description

Implemented test from task 10344 and 10345

## List of changes proposed in this pull request

- added a step to check if the avviso PagoPa is present and anothet step to open it
- added a step to check if the avviso f24 is present and anothet step to open it
- added a step to check if the url contain the string .pdf meaning the file is correctly opened

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@downloadFileNotificaConPagamento"` and verify if the tests work fine.

Run on AWS: https://eu-south-1.console.aws.amazon.com/codesuite/codebuild/151559006927/projects/pn-RunFeE2eTestsCodeBuildProject/build/pn-RunFeE2eTestsCodeBuildProject%3A07c94ba3-04b2-4560-a7bc-f1e26e3b9737?region=eu-south-1
